website/docs/r/is_instance_boot_volume.html.markdown

```diff
@@ -0,0 +1,121 @@
+---
+
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : instance_boot_volume"
+description: |-
+  Manages IBM volume.
+---
+
+# ibm_is_instance_boot_volume
+Provides a resource to manage a boot volume of a VPC virtual server instance. Manage the boot volume of a VPC virtual server instance created alongwith the instance creation with this resource.
+
+~> **NOTE:** This is an advanced resource with special caveats. Please read this document in its entirety before using this resource. The `ibm_is_instance_boot_volume` resource behaves differently from normal resources. Terraform does not _create_ this resource but instead attempts to "adopt" it into management.
+
+Every Virtual server instance has a boot volume that can be managed but not destroyed. When Terraform first adopts a instance_boot_volume, 
```

Comment:
need to rephrase  - VSI has a boot volume that needs to be managed and not destroyed.